### ISSUE TITLE
Popularity/Stars for Referenced Repos

### ISSUE THREAD

**Issue by tracker1** [appcypher/awesome-wasm-runtimes#28]
Would be nice to add a github stars badge (and maybe issues, etc) for each of the referenced github projects... this would allow someone to at least have some understanding of where a project stands in terms of popularity.

AFAIK, Wasmer is one of the most popular, along with Node, but knowing what's out there would be nice.

**Comment by appcypher** [comment]
I started thinking about this again. 
Do you or anyone have suggestions on what service to use to keep the star count up-to-date? 
Ideally a free one. 😅 
I also would prefer not have a server running somewhere.

**Comment by tracker1** [comment]
I've used shields.io before, but there are other options.  You just put the appropriate image code in the readme file.

https://shields.io/category/social

https://shields.io/category/issue-tracking

**Comment by codefromthecrypt** [comment]
ps which stars to count is always interesting in GitHub. A lot of projects try to culture people to aggregate stars to a central one. I've certainly encouraged this in the past, though I would say that stars are relevant to the context of the tech they are used as well. For example, many webassembly runtimes offer bindings in different languages, but to the degree they are used or work .. mileage may vary.

In other words, I would suggest that if things are categorized based on stars that the star relevant to the ecosystem is used. For example, if golang, the golang binding's star count, not the umbrella project. I do have stake in things being measured precisely, so feel free to ignore the feedback!

**Comment by appcypher** [comment]
> In other words, I would suggest that if things are categorized based on stars that the star relevant to the ecosystem is used. For example, if golang, the golang binding's star count, not the umbrella project.

Oh, I have no intention of categorising things based on stars. The list is ordered alphanumerically and it will stay that way until there is something better. The badges, once added, should help provide more context on the project.

Also since the list only refers to the the runtime repos only, I cannot aggregate star count of other repos related to the project. Just the main repos mentioned in the list.

Feedback always welcome. 🙂

**Comment by pannous** [comment]
In addition to stars, an automatic "last commit" would be very helpful because by now many runtimes in the list are DEAD for 4 years!

**Comment by codefromthecrypt** [comment]
@pannous agree on the dead thing, though hopefully runtimes don't accumulate so fast that marking dead ones manually becomes too much work ;)

**Comment by codefromthecrypt** [comment]
one health signal troubling to me is where last commit is recent, but last version tag ancient. it is hard on the ecosystem this way, though forgivable for the first year or two.
